Content.Server/_Goobstation/StationEvents/Metric/Components/DoorMetricComponent.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+
+namespace Content.Server._Goobstation.StationEvents.Metric.Components;
```

Comment:
Using directive is unnecessary
```suggestion
namespace Content.Server._Goobstation.StationEvents.Metric.Components;
```